Intro table updates 4 (#276)
patternfly/training-scenarios: react-components/table-intro/step4.md

For each array item in each row's `cells` array, replace the string value with an object whose `title` property is the row cell value.

> Each array item in each row’s cells array looks like this:

Reviewer: might help with the awkwardness of the sentence

```suggestion
For every array item in each row’s cells array looks like this:
```

Author: Revised this section a bit for clarity, let me know what you think

<pre class="file">
{
  cells: ["Row 1 column 1", "Row 1 column 2", "Row 1 column 3"]
}
</pre>

> After replacing the string value with an object whose title property is the row cell value, the code should look like this:

Reviewer: ```suggestion
After replacing the string value with an object that has a title property equal to the row cell value, the code should look like this:
```
